units/en/unit0/introduction.mdx

```diff
@@ -48,8 +48,6 @@ Don't forget to **<a href="https://bit.ly/hf-learn-agents">sign up to the course
 
 (We are respectful of your privacy. We collect your email address to be able to **send you the links when each Unit is published and give you information about the challenges and updates).**
 
-Sign up  👉 <a href="https://bit.ly/hf-learn-agents">here</a>
-
```

Comment:
Why remove this? Even if the content is public, students could want to sign up to get involved with the activities and study groups, get the certification, etc.